qutebrowser/browser/downloads.py

```diff
@@ -727,6 +736,23 @@ def _after_create_parent_question(self,
         else:
             self._after_set_filename()
 
+    def _get_conflicting_downloads(self):
+        """Returns a list of other active downloads with the same name."""
+        return [
+            download for download in
+            objreg.get('download-model', scope='window',
+                       window='current')._all_downloads()
+            if download is not self and
+            download._filename == self._filename and
+            not download.done
+        ]
```

Comment:
Shouldn't there be only one conflicting download?

Reply:
Ideally yes, but the code doesn't necessarily change that much if we look for only one download instead of all of them. You think we should `break` after finding one occurrence and return a single item (or `None` if there aren't any)?

Reply:
Not important but the simpler the better.

Reply:
Now that I revisited this I had a chance to include your suggestion. Thanks :+1: